docs/content/en/docs/contribution-guide/development-setup.md

```diff
@@ -132,6 +132,28 @@ Verify that Tetragon is installed by running:
 kubectl get pods -n kube-system
 ```
 
+### Running Tetragon via Helm
```

Comment:
It's not clear to me what the purpose of these changes is. Could you please provide some context?

This page mentions local development setup, but the instructions below do not seem to be about local development.

Also, the instructions that are removed in `DEVELOP.md` do not seem be to added elsewhere or replaced with something else.

Reply:
from ```DEVELOP.md``` I have not included starting k8s cluster as in this page its already mentioned above with ```kind-setup``` . for local setup developer can make changes to the code and by this guide he can build images of those latest changes and deploy smoothly via helm if he prefers helm option

Reply:
Still I would like to know what changes you want next . thanks

Reply:
So what is covered by this section that is not covered by the kind section above?

Reply:
Ya since kind-setup does the same I think we might end up changing this pr to just add ```export DOCKER_IMAGE_TAG=v1.0.0``` on the kind section

Reply:
What I think might be useful to document is how to develop the Helm chart, e.g. how to pass custom values or upgrade. Not necessarily here, but somewhere in the contribution guide. Currently the instructions here use the published Helm instead of the local one, the developer doc should rather use make commands or `install-tetragon.sh` script I think.

Reply:
could you please conclude what needs to be done in this pr @lambdanis  ?

Reply:
@sadath-12 The idea is to document how to develop the Helm chart: installation using the local Helm chart and passing custom values.